learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -125,17 +132,19 @@ Once they are displayed, connect the "pressed" signal:
 
 .. image:: img/visual_script10.png
 
-This will open the connection dialog. In this dialog, we must select the node where the signal be
+
+This will open the connection dialog. In this dialog, we must select the node where the signal will be
 connected to, and the function that will receive the signal:
```

Comment:
There's a mix of second person ("you") and first person plural ("we") in this doc. We should stick to one - I prefer second person, but am open to debate.